requirements/requirements.txt

```diff
@@ -26,7 +26,6 @@ gunicorn==18.0
 lxml==3.4.4
 kafka-python==0.9.4
 mock==0.8.0
```

Comment:
lol! I just did a quick grep to see if it was imported anywhere. Was trying to not get rabbit-holed in this PR, so didn't go any further than that. Guess I'll have to do more digging/refactoring.